script.js

```diff
@@ -0,0 +1,9 @@
+// var aladin = A.aladin('#aladin-lite-div', {
+//     target: 'helix',
+//     // survey: "", //find survey name
+//     fov: 1
+// });
+
+var aladin = A.aladin('#aladin-lite-div', {target: 'Crab', fov: 50});
+
+aladin.addCatalog(A.catalogFromVizieR('J/ApJS/218/23/table4','Crab', 10, {onClick: 'showTable'}));
```

Comment:
Please split out the config across more lines so that it's more readable.
(auto-format JS file in Atom)